applications.yaml

```diff
@@ -173,6 +173,17 @@
   options:
   - border_overflow
   - tray_and_multi_window
+- name: DBeaver
+  identifier:
+    kind: Exe
+    id: dbeaver.exe
+    matching_strategy: Equals
+  float_identifiers:
+  - kind: Class
+    id: '#32770'
+    comment: All popups appear to use this
+  options:
+  - border_overflow
```

Comment:
DBeaver window does not seem to border overflow for me:
![image](https://github.com/LGUG2Z/komorebi-application-specific-configuration/assets/56180050/d4498d9a-e352-42fa-8dc7-c5b9f0b3a3b4)

Reply:
I have `active_window_border` set to true. Yesterday that border was being overlapped by the app. Today on the first load it overlapped, but then when I moved the window to make it smaller for a screenshot, it corrected the border. I tried loading the app a few times afterwards and the border didn't overlap. There is probably a different issue I'm experiencing. I've removed that option.